esp-hal/src/embassy/executor/interrupt.rs

```diff
@@ -10,6 +10,9 @@ use crate::{
     system::SoftwareInterrupt,
 };
 
+/// The number to offset SW interrupts when passed through the pender context.
+pub const SW_OFFSET: u8 = 16;
```

Comment:
Plucked from thin air, this has no meaning I just chose 16 at random.

Reply:
Why is this needed, why not change the thread-mode from `0` to `4` instead?